tests/test_attribute_definition.py

```diff
@@ -618,10 +618,38 @@ class ParamAttrDefAttr(ParametrizedAttribute):
     arg1: ParameterDef[Attribute]
     arg2: ParameterDef[BoolData]
 
+    # Check that we can defined methods
+    def test(self):
+        pass
```

Comment:
I changed a bit the comment.
It is testing that you can define functions in attributes/operations.
Since methods are also fields, we need to make sure that we can define them without trigerring an error.